os/mm/mm_heap/mm_heapinfo.c

```diff
@@ -112,6 +112,14 @@ void heapinfo_parse(FAR struct mm_heap_s *heap, int mode, pid_t pid)
 #else
 #define region 0
 #endif
+
+#ifdef CONFIG_DEBUG_CHECK_FRAGMENTATION
+	int ndx;
+	int free_nodelist_cnt[MM_NNODES] = {0, };
+	size_t free_nodelist_size[MM_NNODES] = {0, };
```

Comment:
They look freed nodelist count and size.
How about `nodelist_free_cnt` or `free_node_cnt`?

Reply:
OK. I renamed those without 'free_' prefix.